Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.py

```diff
@@ -467,6 +588,115 @@ def get_users_command(client):
     )
 
 
+def get_users_preference_command(client):
+    result = client.get_users_preference()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    del result['success']
+    table_header = list(result.keys())
+    display_title = "User's Preference"
+    markdown = tableToMarkdown(display_title, result, headers=table_header)
```

Comment:
```suggestion
    if not result or len(result) == 0:
        markdown = 'No users preference found.'
    else:
        markdown = tableToMarkdown(display_title, result, headers=table_header)
```

Better to return human readable message instead of empty table., please do the same in the methods:
- get_logpoints_command
- get_repos_command
- get_devices_command
- get_livesearches_command
- search_logs_command

Reply:
Please note that we have added the suggested changes in this [5979a35](https://github.com/demisto/content/pull/12790/commits/5979a3574ba3cadbd9ae7bcffb9d642ecc0728b7) commit.

Reply:
#### Splitting `lp-search-logs` command

Log searching steps in LogPoint Search API:

1. Send `query`, `limit`, `repos`, `time_range` in `/getsearchlogs` endpoint. Respone for this will be a `search_id` and other information (not so relevant in this case).
2. Send the `search_id` in `/getsearchlogs` endpoint. Response will be the search result (logs)

If we split those two requests into two commands, users have to get the `search_id` and then use it as another commands argument in order to get search results. 
We found this not so user friendly and decided to merge into a single command to get search results quickly.

Please give your feedbacks regarding this.

Reply:
I agree with you that one command is more user friendly but its better to have 2 commands, more easy to debug and more much close to the product API.
The customer will use this commands in playbooks and if the command failed for some reason it will be much easier to understand why.

Reply:
Hi @adi88d,

We have now made two separate commands `lp-get-searchid` and `lp-search-logs` for searching logs as requested. We have reviewed all the suggestions and made necessary changes accordingly. Please go ahead with the review process.